examples/tutorial/hello/node/README.md

```diff
@@ -1,83 +1,178 @@
-# Tutorial 1: NodeJS Function w/ Input (3 minutes)
+# Node Function Hello World
 
-This example will show you how to test and deploy Node code to Fn. It will also demonstrate passing data in through stdin.
+This example shows you how to test and deploy Node code to Fn. It also demonstrates passing JSON data to your function through `stdin`. 
 
-### First, run the following commands:
+## Before you Begin
 
-```sh
-# Initialize your function creating a func.yaml file
-fn init --name hello-node
+This tutorial assumes you have installed Docker, Fn server, and Fn CLI.
 
-# Test your function. 
-# This will run inside a container exactly how it will on the server. It will also install and vendor dependencies from Gemfile
-fn run
+## Start Fn Server
 
-# Now try with an input
-cat sample.payload.json | fn run
+Start up the Fn server so we can deploy our function.
 
-# Deploy your functions to the Fn server (default localhost:8080)
-# This will create a route to your function as well
-fn deploy --app myapp
+>```sh
+>fn start
+>```
+
+The command starts Fn in single server mode using an embedded database and message queue. You can find all the
+configuration options [here](../../../../docs/operating/options.md). 
+
+## Create your Function 
+
+* Change into the directory where you want to create your function.
+* Create a `func.js` file.
+* Add the following source code and save the file.
+
+```js
+name = "World";
```

Comment:
You may want to keep it even simpler just like the Java tutorial and not pass in a JSON payload?    Here's a non-JSON version https://github.com/fnproject/tutorials/blob/master/ContainerAsFunction/func.js   Of course being able to easily consume JSON is a nice feature of JavaScript.